com.varnamproject.Varnam.json

```diff
@@ -0,0 +1,171 @@
+{
+    "app-id": "com.varnamproject.Varnam",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "varnam-wrapper",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--filesystem=~/.local/share/varnam"
```

Comment:
```suggestion
```
you should have access to XDG_DATA_DIR already

Reply:
From [the doc](https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories) I understand `XDG_DATA_DIR` is a different directory in `~/.var`. We're using this folder specifically because the word-learnings and training made by Varnam Flatpak app can be used in host system if there's varnam installed in host.

Why we do this is because it's not possible to package an ibus engine in flatpak : https://github.com/flatpak/flatpak/issues/3019